systemd-dissect: add page (#11414)
tldr-pages/tldr: pages/linux/systemd-dissect.md

# systemd-dissect

> > Tool for introspecting and interacting with file system OS disk images, specifically Discoverable Disk Images (DDIs).

Reviewer: ```suggestion
> Introspect and interact with file system OS disk images, specifically Discoverable Disk Images (DDIs).
```

> More information: <https://www.freedesktop.org/software/systemd/man/latest/systemd-dissect.html>.

- Show general image information about the OS image:

`systemd-dissect {{path/to/image.raw}}`

- Mount an OS image:

`systemd-dissect --mount {{path/to/image.raw}} {{/mnt/image}}`

- Unmount an OS image:

`systemd-dissect --umount {{/mnt/image}}`

- List files in an image:

> `systemd-dissect --list {{myimage.raw}}`

Reviewer 2: ```suggestion
`systemd-dissect --list {{path/to/image.raw}}`
```

> - Attach an OS image:
> 
> `systemd-dissect --attach {{path/to/image.raw}}`

Reviewer: ```suggestion
- Attach an OS image to an automatically allocated loopback block device and print its path:

`systemd-dissect --attach {{path/to/image.raw}}`

- Detach an OS image from a loopback block device:

`systemd-dissect --detach {{path/to/image.raw}}`
```

# systemd-dissect

> 
> 

Reviewer 2: ```suggestion

```

One empty line less :P

> Tool for introspecting and interacting with file system OS disk images, specifically Discoverable Disk Images (DDIs).
> More information: <https://www.freedesktop.org/software/systemd/man/latest/systemd-dissect.html>.

> - Show General Image Information about the OS image:

Reviewer 2: ```suggestion
- Show general image information about the OS image:
```

> `systemd-dissect {{myimage.raw}}`

Reviewer 2: ```suggestion
`systemd-dissect {{path/to/image.raw}}`
```
Assuming a relative path is allowed here.

> - Mount an OS Image:

Reviewer 2: ```suggestion
- Mount an OS image:
```

> `systemd-dissect --mount {{myimage.raw}} {{/mnt/myimage}}`

Reviewer 2: ```suggestion
`systemd-dissect --mount {{path/to/image.raw}} {{/mnt/image}}`
```

> - Unmount an OS Image:
> 
> `systemd-dissect --umount {{/mnt/myimage}}`

Reviewer 2: ```suggestion
- Unmount an OS image:

`systemd-dissect --umount {{/mnt/image}}`
```

> - List Files in an Image:
> 
> `systemd-dissect --list {{myimage.raw}}`

Reviewer 2: ```suggestion
- List Files in an image:

`systemd-dissect --list {{path/to/image.raw}}`
```

> - Attach an OS Image:
> 
> `systemd-dissect --attach {{myimage.raw}}`

Reviewer 2: ```suggestion
- Attach an OS image:

`systemd-dissect --attach {{path/to/image.raw}}`
```
